molgenis-core-ui/src/main/resources/js/molgenis.js

```diff
@@ -63,7 +63,10 @@
         $.each(alerts, function (i, alert) {
             if (i > 0)
                 items.push('<br/>');
-            items.push('<span>' + alert.message + '</span>');
+            var message = '<span>' + alert.message;
+            if (alert.code) message += ' (' + alert.code + ')';
+            message += '</span>';
+            items.push(message)
```

Comment:
The open PR from Dennis html escapes this, but I think that breaks the importer exception messages... Leave unescaped for now?

Reply:
We also still have open issue if we want to show it at all.